src/sentry/static/sentry/app/views/groupActivity/noteInput.jsx

```diff
@@ -70,6 +70,11 @@ var NoteInput = React.createClass({
     this.setState({value: e.target.value});
   },
 
+  onKeyDown(e) {
+    // Auto submit the form on [meta] + Enter
+    e.keyCode === 13 && e.metaKey && this.onSubmit(e)
```

Comment:
I think React guarantees a `key` string property on the event object through which you can do:

```
e.key === 'Enter' && e.metaKey && this.onSubmit(e)
```

> Seems kinda hacky since it's passing along the event object that wasn't intended for a submit.

Legitimate concern. I'd create a `submit` method and have both `onKeyDown` and `onSubmit` call that (although having `onSubmit` call `submit` might be confusing).

> Also, fwiw, I originally did e.target.form.submit() like normal JavaScript people would do, and the page reloaded and the form never submit. So I assume that doesn't work for some reason with React.

Yeah, not entirely sure.